src/jsifier.js

```diff
@@ -182,6 +182,9 @@ function JSify(data, functionsOnly) {
           } else if (VERBOSE || WARN_ON_UNDEFINED_SYMBOLS) {
             warn(msg);
           }
+          if (ident === 'main' && STANDALONE_WASM) {
+            warn('To build in STANDALONE_WASM mode without a main(), use emcc --no-entry');
```

Comment:
Or remove `main` from EXPORTED_FUNCTIONS?   .. or coming soon:  pass `-mexec-model=reactor` to clang?   I guess we can update this message as we go.

Reply:
Interesting.. does the abort error() not cause the process to fail?   (I guess not).

Reply:
Yeah, I admit it's more work to update this as the model changes. Still, I'm willing to do that work, seems worth it to help users quickly unbreak.

I think `error()` in the JS compiler prints an error, and maybe also change the exit code, but it doesn't actually halt.